releasenotes/notes/2.0/remove_classical_function-7d9057bc5c77d268.yaml

```
@@ -1,7 +1,7 @@
 ---
 upgrade_circuits:
   - |
-    The :mod:`.classical_function` was removed. This module was dependent on the
-    `tweedledum` library which is no longer compatible with newer versions of Python.
+    The :mod:`.classical_function` module has been removed. This module was dependent on the
```

If it has been removed, should we still mention it with the `:mod:` label? If not I suggest we just use the double tilde. It also seems we're referring to the wrong module? The correct path should be `qiskit.circuit.classicalfunction`:
```suggestion
    The ``qiskit.circuit.classicalfunction`` module has been removed. This module was dependent on the
```